PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs

```diff
@@ -269,6 +269,15 @@ public Collection<Variable> GetScopedVariable()
 
                     if (psVar != null)
                     {
+                        if (psVar.Value is PSObject &&
+                            !(((PSObject)psVar.Value).ImmediateBaseObject is PSCustomObject))
+                        {

```

Comment:
Do we ever have to worry about further nesting?  eg, can the immediateBaseObject be another PSObject?  What if things are piped around and minuplated?  Or is it only ever one layer deep?